runtime/autoload/man.vim

```diff
@@ -1,6 +1,11 @@
-let s:man_tag_depth = 0
+" Ensure Vim is not recursively invoked (man-db does this)
+" More info here http://comments.gmane.org/gmane.editors.vim.devel/29085
+let s:man_cmd = 'unset MANPAGER; man 2>/dev/null '
```

Comment:
@justinmk we could leave it like this, and then change it when #3265 is implemented. Its a few lines, nothing major.